doc/source/myrocks/zenfs.rst

```diff
@@ -13,99 +13,191 @@ a Linux kernel implementation which supports NVMe Zoned Namespaces. The kernel
-
-
-#. Run ``/zenfs mkfs --zbd <short_block_device_name> --aux_path=<path_to_aux_zenfs_dir>`` in the context of the mysql:mysql account to initialize ZenFS on the ZBD.
-
-    .. sourcecode:: bash
-
-        zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux
-
-    .. note::
-
-        This step initializes the metadata on the block device, ``<block_device_name>``, and creates the ``<path_to_aux_zenfs_dir>`` owned by the `mysql:mysql` account. For example, the ``<block_device_name>`` could be ``/dev/nvme1n2``. The ``<path_to_aux_zenfs_dir>`` is a unique, dedicated directory which allows read and write access to `mysql:mysql`.
-
-        If you are re-initializing an already initialized ZBD, you must manually clear the ``/var/lib/mysql_aux`` directory and add the additional ``--force`` parameter.
-
-        .. sourcecode:: bash
-
-            zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux  --force
-
-
+   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For the purposes of this document the ``block_device_name`` is ``/dev/nvme0n2`` and the short name is ``nvme0n2``. 
+    
+The primary supported platform is `Debian 11.1 <https://www.debian.org/releases/bullseye/debian-installer/index>`__.
 
-#. You can start Percona Server for MySQL with any of the known methods. For example, add this option to ``my.cnf``:
 
-    .. sourcecode:: text
+.. _zenfs-installation:
 
-        [mysqld]
-        ...
-        rocksdb-fs-uri=zenfs://dev:nvme1n2
-        ...
+Installation
+============================================================
 
-You can verify if the ZenFS was successfully created with the following command:
+Start with the installation of *Percona Server for MySQL*. 
 
-.. sourcecode:: bash
+1. The steps are listed here for convenience, for an explanation, see :ref:`apt-install`.
 
-    zenfs ls-uuid
-    ...
-    13e421af-1967-435c-ab15-faf4529710b6    nvme1n2
-    ...
 
-You can check the available storage with the following command:
+   .. code-block:: bash
 
-.. sourcecode:: bash
+      $ sudo apt install gnupg2	
+      $ wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo percona-release setup ps80
 
-    zenfs df --zbd=nvme1n2
-    Free: 7563 MB
-    Used: 0 MB
-    Reclaimable: 0 MB
-    Space amplification: 0%
+2. Install the `zenfs` package. The Percona Server for MySQL with MyRocks and the ZenFS plugin package is listed in the :ref:`installing_from_binary_tarball` section of the *Percona Server for MySQL* installation instructions. 
+   
+   .. code-block:: bash
+	
+      $ sudo apt install percona-server-server-zenfs
```

Comment:
Here and everywhere, there will be no Percona Server packages with `-zenfs` suffix. From now on, `percona-server-server`  includes support for ZenFS on selected platforms automatically.

Reply:
Removed -zenfs